vms/administrator/views.py

```diff
@@ -39,6 +39,12 @@ class ShowFormView(AdministratorLoginRequiredMixin, FormView):
     model = Administrator
     form_class = ReportForm
     template_name = "administrator/report.html"
+    job_list = get_jobs_ordered_by_title()
+
+    def get(self, request, *args, **kwargs):
+        return render(request, 'administrator/report.html',
+                  { 'job_list': self.job_list})
+
```

Comment:
@Himanshi-Khandelwal This works good if the user first starts with selecting a job. 

Fails if
1. Admin enters name of the volunteer and then say jobs.
2. Selects an event and then say jobs. 
3. Does anything related and then say jobs. No filtering in such cases. :(